packages/valory/services/oracle_ropsten_cluster/service.yaml

```
@@ -100,9 +100,31 @@ models:
 public_id: valory/ledger:0.1.0
 type: connection
 config:
-  ledger_apis:
-    ethereum:
-      address: https://ropsten.infura.io/v3/2980beeca3544c9fbace4f24218afcd4
-      chain_id: 3
-      poa_chain: false
-      default_gas_price_strategy: eip1559
+  0:
+  - ledger_apis:
+      ethereum:
+        address: ${SERVICE_ORACLE_POLYGON_RPC_0:str:http://127.0.0.1:8545}
+        chain_id: 137
+        poa_chain: true
+        default_gas_price_strategy: eip1559_polygon
```

```suggestion
        poa_chain: false
        default_gas_price_strategy: eip1559
```
needs fixing throughout this file